knative-eventing-extension/README.md

```diff
@@ -0,0 +1,2 @@
+# experimental
+Knative Eventing Extension
```

Comment:
is it possible to add some more initial information on what one would expect to find in this folder and what the plans are?

Reply:
You bet! I've added in a more complete project intro and tied into the community page.

Reply:
excellent! thanks @skaegi :heart: :D